app/models/question_auto_close.rb

```diff
@@ -0,0 +1,111 @@
+# frozen_string_literal: true
+
+class QuestionAutoClose < ApplicationRecord
```

Comment:
```suggestion
class QuestionAutoClose
```

対応するDBテーブルがないのであればApplicationRecordを継承せず、POROで大丈夫です。